src/Powercord/plugins/pc-emojiUtility/index.js

```diff
@@ -133,7 +168,286 @@ module.exports = class EmojiUtility extends Plugin {
     };
   }
 
+  hasPermission (guildId, permission) {
+    const permissions = getGuildPermissions(guildId);
+
+    return permissions && (permissions & permission) !== 0;
+  }
+
+  createFakeEmoji (id, name, url) {
+    return {
+      id,
+      name,
+      url,
+      fake: true
+    };
+  }
+
   start () {
+    this.loadCSS(resolve(__dirname, 'style.scss'));
+
+    const outer = this; // eslint-disable-line consistent-this
```

Comment:
should just be `const _this = this;` (https://github.com/aetheryx/powercord/blob/v2/.eslintrc.json#L145)